x-pack/plugins/ml/common/types/trained_models.ts

```diff
@@ -245,6 +245,10 @@ export interface NodesOverviewResponse {
   nodes: NodeDeploymentStatsResponse[];
 }
 
+export interface SimulateIngestPipelineResponse {
```

Comment:
`estypes` has `IngestSimulateResponse`, maybe we can use that throughout instead of defining our own type here?

Reply:
Good catch! Updated in 4b1791c5523f98483b24f0ca0dd3f1e88b855a50